src/components/EmojiSuggestions/EmojiSuggestions.scss

```diff
@@ -8,7 +8,7 @@
     width: 100%;
     border-radius: $rounded--default;
     background-color: $color-lighter-gray;
-    max-height: min($emoji-suggestions__max-height, calc(100svh - var(--approx-top-distance, 0px)));
+    max-height: min($emoji-suggestions__max-height, calc(100svh - var(--top-distance, 0) * 1px - 2 * $spacing--lg));
```

Comment:
1. why is `svh` used instead of `vh`?
2. why is the value multiplied by 1?

Reply:
`svh` was the first thing I used to make sure it won't overflow, but `vh` will probably work just as well since the user will have to scroll the emoji suggestions into view anyways.

I multiplied it with `1px` because in the component, I set the css variable as a number, without the pixel unit. I realise now, would probably be cleaner to put the unit in the component.

Unfortunately I'm traveling this week and can't really edit these small changes until next Monday.